nvaccess/nvda
# Add-on store: Make add-on version contextual on views

### Link to issue number:
Part of #13985 

### Summary of the issue:
When viewing installed or disabled add-ons, the user wants to know the information on the version of the installed add-on.
When viewing updatable or available add-ons, the user wants to know the information about the available version.

Before this PR, the data show was the available version if it exists, else the installed version.

### Description of user facing changes
When viewing installed or disabled add-ons, the version listed should be the installed version, not the latest version available in the add-on store.

### Description of development approach
Group add-ons by available/installed.
Show the correct add-on list depending on the view.

### Testing strategy:
**Manual testing:**
- with an installed add-on that has an available update, ensure that the version listed in "installed"/"disabled" views is older than the version listed in "available/update"

### Known issues with pull request:
None

### Change log entries:
N/A

### Code Review Checklist:



  - description is up to date
  - change log entries
  - Unit tests
  - System (end to end) tests
  - Manual testing
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
